QgisModelBaker/libqgsprojectgen/dataobjects/project.py

```diff
@@ -115,13 +118,19 @@ def create(self, path: str, qgis_project: QgsProject):
             referencing_field_constraints = referencing_layer.fieldConstraints(rel.referencingFields()[0])
             allow_null = not bool(referencing_field_constraints & QgsFieldConstraints.ConstraintNotNull)
 
+            # If we have an extended ili2db domain, we need to filter its values
+            filter_expression = "\"{}\" = '{}'".format(ENUM_THIS_CLASS_COLUMN,
```

Comment:
Do we really need a constant here? It will be never used again, or am I wrong?

Reply:
Not strictly necessary, just thought about a config file with ili2db common names like "thisclass" so that they can be easily spotted and adjusted.

But it's not this PR's bussiness anyway. If you want we can remove the constant here.

Reply:
Hm... Okay, I see the point. You can keep it like this.